hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/adapters/IncompatibleMutationAdapter.java

```diff
@@ -56,6 +57,11 @@ private void incrementIncompatibleMutations() {
     metricsExporter.incCounters(INCOMPATIBLE_MUTATION_METRIC_KEY, 1);
   }
 
+  private void incrementFutureDeleteMutations() {
+    metricsExporter.incCounters(FUTURE_DELETE_MUTATION_METRIC_KEY, 1);
```

Comment:
Like we discussed yesterday, this is just informational. This is not an incompatible mutation. Please do not increment the incompatible mutation metric here.

Reply:
Done.